packages/vite/src/node/plugins/css.ts

```diff
@@ -556,16 +557,20 @@ export function cssPostPlugin(config: ResolvedConfig): Plugin {
           })
           chunk.viteMetadata.importedCss.add(this.getFileName(fileHandle))
         } else if (!config.build.ssr) {
-          // legacy build and inline css
-
-          // __VITE_ASSET__ and __VITE_PUBLIC_ASSET__ urls are processed by
-          // the vite:asset plugin, don't call resolveAssetUrlsInCss here
+          chunkCSS =
+            renderAssetUrlInJS(
+              this,
+              config,
+              chunk,
+              opts,
+              chunkCSS,
+              '`'
+            )?.toString() || chunkCSS
           chunkCSS = await finalizeCss(chunkCSS, true, config)
```

Comment:
We can do it like this, and then we don't need to wrap it
```suggestion
          chunkCSS = await finalizeCss(chunkCSS, true, config)
          let cssString = JSON.stringify(chunkCSS)
            renderAssetUrlInJS(
              this,
              config,
              chunk,
              opts,
              cssString,
              '`'
            )?.toString() || cssString
```

Reply:
wow, nice!!!

Reply:
Oh, I see now that you have the config to make the template string work. Still, I think it is better to keep using " here. For reference, we were discussing about a problem related to this yesterday https://github.com/vitejs/vite/issues/9694#issuecomment-1216294243

Reply:
yes, it can reduce a params